cmd/kubelet/app/server.go

```diff
@@ -216,7 +216,7 @@ func NewKubeletServer() *KubeletServer {
 		RootDirectory:                  defaultRootDir,
 		SerializeImagePulls:            true,
 		StreamingConnectionIdleTimeout: 5 * time.Minute,
-		SyncFrequency:                  10 * time.Second,
+		SyncFrequency:                  1 * time.Minute,
```

Comment:
Seems like a pretty big change that is not strictly related to the point of this PR... am I missing something?